app/api/groups/groups.go

```diff
@@ -43,6 +43,7 @@ func (srv *Service) SetRoutes(router chi.Router) {
 	router.Post("/groups/{parent_group_id}/requests/reject", service.AppHandler(srv.rejectRequests).ServeHTTP)
 
 	router.Post("/groups/{parent_group_id}/invitations", service.AppHandler(srv.inviteUsers).ServeHTTP)
+	router.Post("/groups/{parent_group_id}/invitations/withdraw", service.AppHandler(srv.withdrawInvitations).ServeHTTP)
```

Comment:
We should have done that with a DELETE.  Unlike accept/refuse, "withdraw" is really just a delete. 
(maybe not great for mass operations anyway)